aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts

```diff
@@ -0,0 +1,12 @@
+import { makeStyles } from '@material-ui/core';
+
+export default makeStyles((theme) => ({
+  popover: {
+    pointerEvents: 'none',
+  },
+  popoverContent: {
+    margin: '1em',
+    width: 250,
+    height: 60,
+  },
+}));
```

Comment:
I think we should start writing styles in scss files

Reply:
did you refactor all the styles?

Reply:
No, but the new styles I writing in scss files.

Reply:
ok i will fix this, but for all project we can refactor it in one task